src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -118,7 +118,7 @@
         fsyncWarningThresholdMS = fsyncWarningThreshold;
     }
 
-    long lastZxidSeen;
+    long lastMaxZxidSeen;
```

Comment:
oh, last minute change left the issue here, was thinking of lastMaxZxidSeen will be more accurate, but thought a bit more and lastZxidSeen should be Ok too, so changed it back, forgot to change here.